### PR TITLE
clarify usage of `IntMap EventType` in `ppEvent`

### PR DESCRIPTION
**Pull request by adamse**

### CODE REVIEW

File: src/GHC/RTS/Events.hs [hunk 1]
```diff
@@ -621,7 +621,10 @@ buildEventType (EventType num dsc msz) =
   <> maybe "variable" TB.decimal msz <> ")"
 
 -- | Pretty prints an 'Event', with clean handling for 'UnknownEvent'
-ppEvent :: IntMap EventType -> Event -> String
+ppEvent 
+  :: IntMap EventType -- ^ Look up @'UnknownEvent'.'ref'@ to find a suitable description.
```
Comment (Mikolaj) [inline src/GHC/RTS/Events.hs:625]: What does this Haddock syntax do, visually and link-wise?

Comment (adamse) [reply src/GHC/RTS/Events.hs:625]: image

Comment (adamse) [reply src/GHC/RTS/Events.hs:625]: linking to where you would expect

Comment (Mikolaj) [reply src/GHC/RTS/Events.hs:625]: TIL. And what happens when you remove the '@'?

**Review by Mikolaj — APPROVED** [review]
LGTM.

**Comment by maoe** [comment]
LGTM

**Pull request merged**